### PR TITLE
Add recipe for mlpy

### PR DESCRIPTION
**Pull request by npavlovikj**

### CODE REVIEW

**Comment by npavlovikj** [comment]
@conda-forge/help-python , can someone please help me with the Windows build? It fails with `LINK : fatal error LNK1181: cannot open input file 'm.lib'`. 
I wonder if this package supports win32 only, and not win64 (https://github.com/richardARPANET/mlpy/blob/509bc3659d079c4050e1763b33e587717f9ca51a/setup.py), but I am not proficient enough to determine that. 
I would really appreciate if someone can take a look and give me some tips.

**Comment by npavlovikj** [comment]
@conda-forge/staged-recipes , please review.

File: recipes/machine-learning-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - python <3.7
+    - setuptools
+    - numpy >=1.3.0
+    - scipy >=0.7.0
+    - gsl >=1.11
+  run:
+    - python <3.7
+    - setuptools
```
Comment (scopatz) [inline recipes/machine-learning-py/meta.yaml]: Does this really depend on setuptools at runtime?

**Comment by npavlovikj** [comment]
Fixed @scopatz  :)

**Comment by scopatz** [comment]
Thanks @npavlovikj!

**Comment by npavlovikj** [comment]
@scopatz , this feedstock is not yet created. Do you know what causes the delay?

**Comment by scopatz** [comment]
I think all feedstock creation is backed up right now.  Not sure exactly why.  You can try asking on gitter

**Comment by npavlovikj** [comment]
Thanks Anthony, will check there.

**Pull request merged**